heracles/fields.py

```diff
@@ -305,10 +305,14 @@ async def __call__(
             del vis
 
         # compute bias of positions, including weight variance
-        bias = ngal / (4 * np.pi) * mapper.area**2 * (w2mean / nbar**2)
+        variance = w2mean / wmean**2
+        neff = (npix * nbar) / (4 * np.pi * wmean)
```

Comment:
I think for positions this needs to be
```suggestion
        variance = 1.0
        neff = (nbar / mapper.area)**2 / (ngal / (4 * np.pi * fsky)) / w2mean
```

Reply:
I don't think I understand these last two suggestions.
In the Scalar and complex fields we don't define w2mean. 
Instead we have wmean and var.